Apps/WebClient/src/ClientApp/test/utility/stubUtil.ts

```diff
@@ -0,0 +1,9 @@
+export const stubbedPromise = <T = void>(): Promise<T> => {
```

Comment:
since these are stubbed methods; What do you think about moving them inside the stub folder
```test/stubs/util```